lib/result-model.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
```

Comment:
Node 4 is the earliest version we could support with es6 features but we need to use `'use strict';` for it to work